panic on renew

Attempting to renew an existing domain using the wrong account email results in a panic:
```sh
./lego --email=baduser@example.com --path=/home/letsencrypt/store -d example.com --webroot=/var/www/letsencrypt renew
2017/10/25 13:40:24 [INFO][example.com] acme: Trying renewal with 2158 hours remaining
2017/10/25 13:40:24 [INFO][example.com, www.example.com] acme: Obtaining bundled SAN certificate
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xc8 pc=0x789c82]

goroutine 15 [running]:
github.com/xenolf/lego/acme.(*Client).getChallenges.func1(0xc42046e0e0, 0xc420054ba0, 0xc420054b40, 0xc4201a0800, 0xa)
        /mnt/d/Programming/GoPath/src/github.com/xenolf/lego/acme/client.go:550 +0x132
created by github.com/xenolf/lego/acme.(*Client).getChallenges
        /mnt/d/Programming/GoPath/src/github.com/xenolf/lego/acme/client.go:547 +0x133
```
(the actual domain + email are different to the above)

Using v0.4.1

--- a/cli_handlers.go
+++ b/cli_handlers.go
@@ -329,8 +329,10 @@ func run(c *cli.Context) error {
 }
 
 func revoke(c *cli.Context) error {
-
-	conf, _, client := setup(c)
+	conf, acc, client := setup(c)
+	if acc.Registration == nil {
+		logger().Fatalf("Account %s is not registered. Use 'run' to register a new account.\n", acc.Email)
+	}
 
 	err := checkFolder(conf.CertPath())
 	if err != nil {
@@ -355,7 +357,10 @@ func revoke(c *cli.Context) error {
 }
 
 func renew(c *cli.Context) error {
-	conf, _, client := setup(c)
+	conf, acc, client := setup(c)
+	if acc.Registration == nil {
+		logger().Fatalf("Account %s is not registered. Use 'run' to register a new account.\n", acc.Email)
+	}
 
 	if len(c.GlobalStringSlice("domains")) <= 0 {
 		logger().Fatal("Please specify at least one domain.")

